DeskReserve_Tests/DeskServiceTest.cs

```diff
@@ -0,0 +1,111 @@
+﻿using DeskReserve.Data.DBContext.Entity;
+using DeskReserve.Domain;
+using DeskReserve.Exception;
+using DeskReserve.Repository;
+using Moq;
+
+namespace Desk_Reserve.Tests
+{
+	[TestFixture]
+	public class DeskServiceTests
+	{
+		[Test]
+		public async Task GetAllAsync_ReturnsAllDesks()
+		{
+			var repositoryMock = new Mock<IDeskRepository>();
```

Comment:
Since we are using the repository for all test try to isolate this in a method that mock our dependencies. Look up TestFixture